frontend/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, SimpleGrid } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { Avatar } from './Avatar'
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+} as Meta
+
+type AvatarTemplateProps = {
+  hasNotification?: boolean
+}
+
+const AvatarTemplate: Story<AvatarTemplateProps> = ({ hasNotification }) => {
+  return <Avatar hasNotification={hasNotification}></Avatar>
+}
+
+const AvatarGroupTemplate: Story = () => {
```

Comment:
should we split the story into two separate stories? Makes it easier to see what are the available props

WithNotification
WithoutNotification

Reply:
thanks! edited